cli/build_run_result_registerer.go

```diff
@@ -0,0 +1,139 @@
+package cli
+
+import (
+	"errors"
+	"time"
+
+	"github.com/bitrise-io/bitrise/analytics"
+	"github.com/bitrise-io/bitrise/bitrise"
+	"github.com/bitrise-io/bitrise/exitcode"
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/bitrise/tools/timeoutcmd"
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/pointers"
+	coreanalytics "github.com/bitrise-io/go-utils/v2/analytics"
```

Comment:
I think it would be better to import this as-is and create an alias for the v1 `analytics`. This way, when we get rid of the v1 imports, we won't have to remove the `v2` import alias and update every affected line.

Reply:
It's not the v1 and v2 versions of the `go-utils/analytics` package, 
but v2 `go-utils/analytics` and `bitrise/analytics` packages.